test/test_linalg.py

```diff
@@ -5127,13 +5127,13 @@ def test_householder_product_errors_and_warnings(self, device):
             # device of out and input should match
             wrong_device = 'cpu' if self.device_type != 'cpu' else 'cuda'
             out = torch.empty_like(reflectors).to(wrong_device)
-            with self.assertRaisesRegex(RuntimeError, "Expected result and input tensors to be on the same device"):
+            with self.assertRaisesRegex(RuntimeError, "Expected all tensors to be on the same device"):
```

Comment:
Since you have manually fixed all of these, no further action is necessary. However, in the future, we should consider making error messages all use expect testing (see `test/test_dispatch.py` for example). These would then be updatable automatically using `EXPECTTEST_ACCEPT` without having to manually update the error strings in this case.